js/demo.js

```diff
@@ -26,6 +27,13 @@ app.controller('SearchController', function($scope, $rootScope, $sce, $http) {
       maxBounds: L.latLngBounds(L.latLng(-80, -180), L.latLng(82, 180))
   });
 
+  if(!$rootScope.geobase.locationProvided && navigator.geolocation){
+    navigator.geolocation.getCurrentPosition(function (pos){
+      var coords = pos.coords;
+      map.panTo(new L.LatLng(coords.latitude, coords.longitude));
```

Comment:
and instead of just panning to the lat/lon, also trigger an event ```new-location``` and pass lat/lon/zoom in an object so, a new $rootScope.geobase is set